drizzle/utils.py

```diff
@@ -61,6 +61,8 @@ def calc_pixmap(wcs_from, wcs_to, shape=None):
         shape = wcs_from.array_shape
         if shape is None:
             if (bbox := getattr(wcs_from, "bounding_box", None)) is not None:
+                if not isinstance(bbox, tuple):
+                    bbox = bbox.bounding_box()
```

Comment:
This will work only if bounding_box is an instance of `modeling.bounding_box.ModelBoundingBox`, so a better check is that it is that instance.